api/src/main/java/org/xwiki/filemanager/internal/job/PackJob.java

```diff
@@ -115,6 +115,7 @@ protected void runInternal() throws Exception
         AttachmentReference outputFileReference = getRequest().getOutputFileReference();
         TemporaryResourceReference temporaryResourceReference = new TemporaryResourceReference(MODULE_NAME,
             Collections.singletonList(outputFileReference.getName()), outputFileReference.getParent());
+        temporaryResourceReference.addParameter("jobId", request.getId().get(1));
```

Comment:
```suggestion
        temporaryResourceReference.addParameter("jobId", StringUtils.join(request.getId(), "/"));
```

I find it safer to use the full id.